app/views/catalog/_members_default.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if resource.respond_to?(:member_ids) %>
+  <h2>Members</h2>
+  <table class="table table-striped">
+    <tr>
+     <th>Thumbnail</th>
+     <th>Title</th>
+     <th>Date Uploaded</th>
+     <th>Visibility</th>
+     <th>Actions</th>
+    </tr>
+    <% if resource.member_ids %>
+      <% resource.member_ids.map { |id| Valkyrie::MetadataAdapter.find(:indexing_persister).query_service.find_by(id: id) }.reject { |r| r.is_a?(FileSet) }.each do |member| %>
```

Comment:
This is pretty gnarly - maybe you can use the decorator and a similar strategy to https://github.com/pulibrary/figgy/blob/master/app/decorators/scanned_resource_decorator.rb#L7 ?